gui/static/js/components/gridview.js

```diff
@@ -148,7 +148,7 @@ export default class GridView extends React.Component {
         //     this.setup(background);
         // }
 
-        console.log("background failed to load! using white background");
+        // console.log("background failed to load! using white background");
```

Comment:
Rather than commenting these out, we can change to a more formal `console.warn()` message or just remove them entirely. Here and below.

Reply:
I'll remove them